src/net/http.rs

```diff
@@ -110,6 +110,7 @@ lazy_static! {
         "^/v2/contracts/interface/(?P<address>{})/(?P<contract>{})$",
         *STANDARD_PRINCIPAL_REGEX, *CONTRACT_NAME_REGEX)).unwrap();
     static ref PATH_GET_TRANSFER_COST: Regex = Regex::new("^/v2/fees/transfer$").unwrap();
+    static ref PATH_OPTIONS_WILDCARD: Regex = Regex::new(".*").unwrap();
```

Comment:
What can this path realistically be?  I don't like having wildcards in input validation code if they can be avoided.

Reply:
The idea is that it matches OPTIONS for any request, but we only really need it for a POST. This could be `^/v2/*`?

Reply:
Can we change it to `^.{0,4096}$` or something to bound the number of characters the regex will try to match?

Reply:
Changed it to limit the length, as well as prefix with `/v2`